### PR TITLE
Fix #7156: Add Sample Data for Kafka and Redpanda

### LINKED ISSUE #7156: Missing sample data for Kafka and Redpanda connector when ingesting metadata
**Affected module**
Ingestion Framework

**Describe the bug**
Kafka and Redpanda do not show SampleData

**To Reproduce**
Ingest metadata for a topic that has data

**Expected behavior**
Show some data in the Sample Data tab

**Version:**
 - OpenMetadata version: 0.11.5

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Fix #7156: Add Sample Data for Kafka and Redpanda

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py [hunk 1]
```diff
@@ -199,37 +195,56 @@ def _parse_topic_metadata(self, topic_name: str) -> Optional[Schema]:
...
+        self, topic_details: BrokerTopicDetails
+    ) -> TopicSampleData:
+        """
+        Method to Get Sample Data of Messaging Entity
+        """
+        if (
+            self.context.topic
+            and self.context.topic.messageSchema
+            and self.context.topic.messageSchema.schemaType.value
+            == SchemaType.Avro.value
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py:208]: Why do we need it to be Avro? 🤔

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py:208]: not sure why that's the case, I just translated [this](https://github.com/open-metadata/OpenMetadata/blob/831edcf0026e6247a660f03ddde9b91b1756b7ae/ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py#L130) logic here 😅

Comment (pmbrull) [reply ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py:208]: not sure if we have a way to test it out with JSON Schema and Protobuf, as we support those type of messages 🤔

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/messaging/common_broker_source.py:208]: yup will check if I can test it with protobuf

**Review by pmbrull — APPROVED** [review]
Let's open a ticket and track protobuf and json schema sample data later. Thanks

**Pull request merged**